File: Ejercicios Propuestos 1/Ejercicio 9/Ejercicio 9.java
import java.util.Scanner;

public class Ejercicio9 {
    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {
            System.out.print("Ingrese el valor inicial (n): ");
            int n = sc.nextInt();
            System.out.print("Ingrese el valor final (m): ");
            int m = sc.nextInt();
            int a = 0, b = 1;
            System.out.println("Serie de Fibonacci entre " + n + " y " + m + ":");
            while (a <= m) {
                if (a >= n) {
                    System.out.print(a + " ");
                }
                int siguiente = a + b;
                a = b;
                b = siguiente;
            }
        }
    }
}

File: Ejercicios Propuestos 1/Ejercicio 9/Ejercicio 9.jl
print("Ingrese el límite inferior (n): ")
n = parse(Int, readline())
print("Ingrese el límite superior (m): ")
m = parse(Int, readline())

if n > m
    temp_intercambio = n
    global n = m
    global m = temp_intercambio
    println("Nota: Se acomodaron los límites automáticamente de menor a mayor.")
end

a = 0
b = 1

println("Los números de Fibonacci entre ", n, " y ", m, " son:")

while a <= m
    if a >= n
        print(a, " ")
    end
    siguiente = a + b
    global a = b
    global b = siguiente
end
println()
